flowauth/frontend/src/ServerAdminDetails.jsx

```
@@ -116,14 +116,14 @@ class ServerAdminDetails extends React.Component {
           )
         : createServer(name, new Date(latest_expiry).toISOString(), max_life);
       try {
-        await editServerCapabilities((await server).id, rightsObjs);
+        await editServerScopes((await server).id, rightsObjs);
         onClick();
       } catch (err) {
-        if (err.code === 400) {
-          this.setState({ pageError: true, errors: err });
-        } else {
-          this.setState({ hasError: true, error: err });
-        }
+        // if (err.code === 400) {
```

Uh-oh, swallowing exceptions is naughty!